README.md

```diff
@@ -154,7 +177,8 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+There is an infinite number of sentences possible in this language. This is due to the fact that an infinite number of adjectives
+can be used, and that adjectives can be repeated. 
```

Comment:
So I had to look this up in a grammar book. 

> There is an infinite number of sentences possible in this language.

The subject of the sentence is "sentences", so it should be "There _are_ an infinite number of sentences...".

However, the word "infinite" can also be used as a collective noun, and if used as a noun it is therefore singular, as in "The number of sentences is infinite."

All of which tells you more about my mental state right now than it is about grading....